doc/source/release-notes/Percona-Server-5.7.28-31.rst

```diff
@@ -0,0 +1,57 @@
+.. rn:: 5.7.28-31:
+
+===============================================================================
+|Percona Server| |release|
+===============================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on |date|. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.28-31/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.27 <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-28.html>`_ and includes all the bug fixes in it. |Percona Server| |release| is now the current GA
+(Generally Available) release in the 5.7 series.
+
+All software developed by Percona is open-source and free.
+
+.. note::
+
+   If you're currently using |Percona Server| 5.7, Percona recommends upgrading to this version of 5.7 prior to upgrading to |Percona Server| 8.0.
+
+Bugs Fixed
+===============================================================================
+
+- The length of time a MySQL query ran increased with a higher number of table partitions. Bug fixed :psbug:`5783`.
```

Comment:
This actually wasn't 'fixed'.  An experimental set of new options were added to prevent deep dive index scans (innodb records_in_range) on all partitions and instead return fixed some values.

Reply:
updated.

Reply:
Should I move this one to Known Issues?